src/Sylius/Bundle/AttributeBundle/Form/Type/AttributeType/Configuration/SelectAttributeConfigurationType.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * Class SelectAttributeConfigurationType
+ * @package Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration
+ */
+class SelectAttributeConfigurationType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('options', 'collection', [
+                'type' => 'text',
```

Comment:
@aramalipoor 
I created a test branch for this feature, you can see it at https://github.com/Lowlo/Sylius/tree/attribute_select_test